src/hotspot/cpu/riscv/assembler_riscv.hpp

```diff
@@ -1074,7 +1074,26 @@ enum operand_size { int8, int16, int32, uint32, int64 };
 
 #undef INSN
 
-// Float and Double Conversion Instruction
+enum FCLASS_MASK {
+  FCLASS_MINF       = 1 << 0,   // negative infinite
+  FCLASS_MNORM      = 1 << 1,   // negative normal number
+  FCLASS_MSUBNORM   = 1 << 2,   // negative subnormal number
+  FCLASS_MZERO      = 1 << 3,   // negative zero
+  FCLASS_PZERO      = 1 << 4,   // positive zero
+  FCLASS_PSUBNORM   = 1 << 5,   // positive subnormal number
+  FCLASS_PNORM      = 1 << 6,   // positive normal number
+  FCLASS_PINF       = 1 << 7,   // positive infinite
+  FCLASS_SNAN       = 1 << 8,   // signaling NaN
+  FCLASS_QNAN       = 1 << 9,   // quiet NaN
+  FCLASS_ZERO       = FCLASS_MZERO    | FCLASS_PZERO,
+  FCLASS_SUBNORM    = FCLASS_MSUBNORM | FCLASS_PSUBNORM,
+  FCLASS_NORM       = FCLASS_MNORM    | FCLASS_PNORM,
+  FCLASS_INF        = FCLASS_MINF     | FCLASS_PINF,
+  FCLASS_NAN        = FCLASS_SNAN     | FCLASS_QNAN,
+  FCLASS_FINITE     = FCLASS_ZERO     | FCLASS_SUBNORM   | FCLASS_NORM,
+};
```

Comment:
We use lower-case and no prefix for similar enums (ex: https://github.com/openjdk/jdk/blob/069bb87693ec7944f08062e3a7e460653324e6cb/src/hotspot/cpu/riscv/assembler_riscv.hpp#L1119-L1145)

```suggestion
enum fclass_mask {
  minf       = 1 << 0,   // negative infinite
  mnorm      = 1 << 1,   // negative normal number
  msubnorm   = 1 << 2,   // negative subnormal number
  mzero      = 1 << 3,   // negative zero
  pzero      = 1 << 4,   // positive zero
  psubnorm   = 1 << 5,   // positive subnormal number
  pnorm      = 1 << 6,   // positive normal number
  pinf       = 1 << 7,   // positive infinite
  snan       = 1 << 8,   // signaling NaN
  qnan       = 1 << 9,   // quiet NaN
  zero       = mzero    | pzero,
  subnorm    = msubnorm | psubnorm,
  norm       = mnorm    | pnorm,
  inf        = minf     | pinf,
  nan        = snan     | qnan,
  finite     = zero     | subnorm   | norm,
};
```

Reply:
Change the enum to lowercase. But I think we should keep the `flcass_` prefix to make sure it‘s only used for `flcass`.